app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java

```diff
@@ -92,12 +92,19 @@ public void prepareView() {
                         imageDetails.getName(),
                         imageDetails.getPath(),
                         editCompoundRecyclerView,
-                        new TagsManager(exifInterface));
+                        new TagsManager(exifInterface),
+                        this);
                 activityEditBinding.setViewModel(editViewModel);
             } catch (IOException e) {
                 Log.d(TAG, String.valueOf(e.getMessage()));
                 //TODO log exception to crashlitycs on else.
             }
         } //TODO log exception to crashlitycs on else.
     }
+
+    @Override
+    public void openEditDialog(Tag tag) {
```

Comment:
Its better to encapsulate interfaces as a private fields. 
Class logic is safe and not exposed for changes from outside.